service/awsconfig/v8/version_bundle.go

```diff
@@ -59,7 +49,7 @@ func VersionBundle() versionbundle.Bundle {
 		Deprecated:   false,
 		Name:         "aws-operator",
 		Time:         time.Date(2018, time.February, 27, 14, 16, 0, 0, time.UTC),
-		Version:      "3.0.0",
-		WIP:          false,
+		Version:      "3.1.0",
```

Comment:
I think the new WIP bundle should be a patch release by default.

Reply:
I don't have opinion. Can be changed any time. I'll change to 3.0.1